### ISSUE TITLE
Configuring user registration form

### ISSUE THREAD

**Issue by deviirmr** [SeaBee-no/documentation#44]
In connection with the Seabee annual meeting in 2023 and the suggestion, we agree to open a user registration page soon. The current GeoNode user registration page has some unknown issues which make the registration form unable to populate in the production environment. Ideally if `ACCOUNT_OPEN_SIGNUP = True` will enable a registration form at GeoNode front-end as per [documentation ](https://docs.geonode.org/en/master/basic/settings/index.html#account-open-signup). 

There are several attempts have been made to track the issue, such as checking the environmental variable configuration at the production server/cluster and debugging the code in the local environment but no success.

Despite  of current situation, there are two possible solution
1. Create a separate/new user registration page (will linked to Djngao authentication)
2. or further attempted to enable the default GeoNode registration page

@knl88 : Do you think we can try the second approach one more time before we dive into the first approach?
@JamesSample

**Comment by knl88** [comment]
Yes we can look into 2) don't think 1) is an option at least not this year.

**Comment by knl88** [comment]
Also suggest that we solve https://github.com/SeaBee-no/helm-deployment/issues/2  so we can invite users a bit easier.
